### ISSUE TITLE
Crash on outoing reaction

### ISSUE THREAD

**Issue by truenicoco** [dino/dino#1343]
Using `0.3.0~git71.20230117.7e0d1db` on fedora 36.

It happened twice to me today, the first time in 1:1 chat; after dino restart I could react just fine.
Then another time in a MUC, this time I could re-crash dino while capturing some stderr/out msgs by trying to react to the latest message of this specific MUC again.
Unlike some of my other reports, the "reaction recipients" were not "bridged entities" but "vanilla XMPP entities" (so it's not my fault ;)).

Both times it felt "slow" to open/select the emoji, don't know if that helps.

```
(dino:121355): Gtk-DEBUG: 13:53:34.066: Found emoji data for en in resource /org/gtk/libgtk/emoji/en.data
(dino:121355): dconf-DEBUG: 13:53:41.406: watch_fast: "/org/gtk/gtk4/settings/emoji-chooser/" (establishing: 0, active: 2)
/home/ncedilni/.local/bin/dino: line 2: 121355 Segmentation fault      (core dumped) GTK_THEME=Adwaita:dark /usr/bin/dino "$@"
```

EDIT: Obtained a gdb thingy: [here](https://paste.sr.ht/~nicoco/90e57ca468c349a27ad1971d033d91b0b660d66a).  `--g-fatal-warnings` prevented dino from lauching within gdb so I did not use this flag.

**Comment by fiaxh** [comment]
This is unfortunately due to a bug in GTK: https://gitlab.gnome.org/GNOME/gtk/-/issues/5190
The bug is already fixed on GTK master and GTK 4.9.1+ alpha versions.

**Comment by FedericoCeratto** [comment]
I can reproduce the the same error with Dino 0.4.0 on Debian Sid.
